hokusai/production.yml

```diff
@@ -118,11 +130,11 @@ spec:
     - port: 80
       protocol: TCP
       name: http
-      targetPort: 8080
+      targetPort: nginx-http
     - port: 443
       protocol: TCP
       name: https
-      targetPort: 8080
+      targetPort: nginx-https
```

Comment:
Wanted to ask for a second look on this one, should they both point at `force-http` instead?

Reply:
Yeah so the ELB is terminating SSL here so it should go to `nginx-http`